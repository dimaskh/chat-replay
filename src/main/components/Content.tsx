// Libs
import React from 'react'

// Components
import Chat from 'modules/chat/components/Chat'
import Player from 'modules/player/containers/Player'
import UsersList from 'modules/users/components/UsersList'

// Styles
import styled from 'styled-components'

export const Content: React.FC = () => (
  <StyledContent>
    <Player />
    <UsersList />
    <Chat />
  </StyledContent>
)

const StyledContent = styled.div`
  display: flex;
  flex-direction: row;
  height: 100%;
  background-color: #282c34;
`

export default Content
