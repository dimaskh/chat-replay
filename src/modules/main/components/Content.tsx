// Libs
import React from 'react'

// Styles
import styled from 'styled-components'

// Components
const Chat = React.lazy(() => import('modules/chat/containers/Chat'))
const Player = React.lazy(() => import('modules/player/containers/Player'))
const UsersList = React.lazy(() => import('modules/users/containers/Users'))

export const Content: React.FC = () => (
  <StyledContent>
    <Player />
    <UsersList />
    <Chat />
  </StyledContent>
)

const StyledContent = styled.div`
  display: flex;
  flex-direction: row;
  height: 100%;
  background-color: #282c34;
`

export default Content
