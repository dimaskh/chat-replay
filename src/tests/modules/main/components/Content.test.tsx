// Libs
import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

// Tested component
import Content from 'modules/main/components/Content'

describe('main/Content component', () => {
  it('should render correctly', () => {
    const component = shallow(<Content />)

    expect(shallowToJson(component)).toMatchSnapshot()
  })
})
