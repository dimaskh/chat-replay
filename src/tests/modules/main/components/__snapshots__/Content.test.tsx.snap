// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main/Content component should render correctly 1`] = `
<styled.div>
  <lazy />
  <lazy />
  <lazy />
</styled.div>
`;
